textalyze.js

```diff
@@ -14,10 +14,32 @@
 function itemCounts(array) {
   let counts = new Map();
 
+  array.forEach(item => {
```

Comment:
Exercise: how would you write this as a single call to [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)?